src/content/language/extension-methods.md

```diff
@@ -197,8 +197,8 @@ to avoid a name conflict when invoking an extension explicitly.
 Use the following syntax to create an extension:
 
 ```plaintext
-extension <extension name>? on <type> {
-  (<member definition>)*
+extension <extension name> on <type> { // extension-name is optional
+  (<member definition>) // can provide one or more member-definitions
```

Comment:
Can we keep the original symbols as well as the comments? Otherwise to those that are familiar with the syntax or skim past the comments, it might look like only one member definition is allowed.

With both the syntax and the comments, all readers are accounted for, plus the readers may become more familiar with the syntax as a result :) 

```suggestion
extension <extension name>? on <type> { // <extension name> is optional.
  (<member definition>)* // Can provide one or more <member definition>.
```